kivy_ios/recipes/jinja2/__init__.py

```diff
@@ -16,10 +16,8 @@ def install(self):
         os.chdir(build_dir)
         hostpython = sh.Command(self.ctx.hostpython)
         build_env = arch.get_env()
-        dest_dir = join(self.ctx.dist_dir, "root", "python")
+        dest_dir = join(self.ctx.dist_dir, "root", "python3")
         build_env['PYTHONPATH'] = join(dest_dir, 'lib', 'python3.8', 'site-packages')
-        cmd = sh.Command("sed")
-        shprint(cmd, "-i", "", "s/setuptools/distutils.core/g", "./setup.py", _env=build_env)
```

Comment:
I would say let's try to see by keeping that workaround if it's enough to fix the current CI build
```
[DEBUG   ] Traceback (most recent call last):
[DEBUG   ]   File "setup.py", line 10, in <module>
[DEBUG   ]     from distutils.core.command.build_ext import build_ext
[DEBUG   ] ModuleNotFoundError: No module named 'distutils.core.command'; 'distutils.core' is not a package
```